pygmt/helpers/testing.py

```diff
@@ -139,3 +140,36 @@ def wrapper(*args, ext="png", request=None, **kwargs):
         return wrapper
 
     return decorator
+
+
+def download_test_data():
+    """
+    Convenience function to download remote data files used in PyGMT tests.
+    """
+    # List of datasets to download
+    datasets = [
+        # Earth relief grids
+        "@earth_relief_01d_p",
+        "@earth_relief_01d_g",
+        "@earth_relief_30m_p",
+        "@earth_relief_30m_g",
+        "@earth_relief_10m_p",
+        "@earth_relief_10m_g",
+        "@earth_relief_05m_p",
+        "@earth_relief_05m_g",
+        # List of tiles of 03s/01s srtm data.
```

Comment:
We cache `01s` data here, no?
```suggestion
        # List of tiles of 03s srtm data.
```